examples/samples/back_projector_proj_matrix_ray_tracing_post_smoothing.par

```diff
@@ -0,0 +1,20 @@
+Back Projector parameters:=
+  
+  type := Matrix
+  Back projector Using Matrix Parameters :=
+    Matrix type := Ray Tracing
+    Ray tracing matrix parameters :=
+    End Ray tracing matrix parameters :=
+  End Back Projector Using Matrix Parameters :=
+
+  Post Data Processor := Separable Cartesian Metz
+  Separable Cartesian Metz Filter Parameters :=
```

Comment:
can we use the Separable Gaussian one instead? Simpler, and no issues with negatives.